packages/stress/src/mode/chat/statsReporter.ts

```diff
@@ -1,57 +1,74 @@
 import type { StressClient } from '../../utils/stressClient'
 import { ChatConfig } from '../common/types'
-import { getLogger } from '../../utils/logger'
 
-const logger = getLogger('stress:statsReporter')
-
-export function statsReporter(rootClient: StressClient, chatConfig: ChatConfig) {
-    let canceled = false
-    let lastReactionCount = 0
-    const interval = setInterval(() => {
-        if (canceled) {
-            return
-        }
-        void (async () => {
-            if (chatConfig.kickoffMessageEventId && chatConfig.countClientsMessageEventId) {
-                const reactionCount = countReactions(
-                    rootClient,
-                    chatConfig.announceChannelId,
-                    chatConfig.kickoffMessageEventId,
-                )
+export function statsReporter(chatConfig: ChatConfig) {
+    return {
+        logStep: (
+            client: StressClient,
+            step: string,
+            isSuccess: boolean,
+            metadata?: Record<string, unknown>,
+        ) => {
+            client.logger.info(
+                {
+                    sequence: 'STRESS_RESULT',
+                    step,
+                    result: isSuccess ? 'PASS' : 'FAIL',
+                    sessionId: client.logId,
```

Comment:
this is the wrong session id - we want all processes to report the same session Id